Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabComponentAdapter.cpp

```diff
@@ -91,4 +104,158 @@ namespace AzToolsFramework::Prefab
         return ComponentAdapter::HandleMessage(message);
     }
 
+    void PrefabComponentAdapter::UpdateDomContents(const PropertyChangeInfo& propertyChangeInfo)
```

Comment:
Right now, this will be triggered only for component property changes. So I think 'propertyChangeInfo' would be more appropriate here. Plus it's not too long of a name. So I'd like to keep it as is for now.